iron-form.html

```diff
@@ -95,6 +95,29 @@
   </template>
 
   <script>
+  (function() {
+    function isSubmittable(node, ignoreName) {
+      return  (!node.disabled && (ignoreName || node.name));
+    }
+
+    function handleSubmittableInSlot(submittable, node, ignoreName) {
+      var assignedNodes = Polymer.dom(node).getDistributedNodes();
+
+      for (var i = 0; i < assignedNodes.length; i++) {
+        if (assignedNodes[i].nodeType === Node.TEXT_NODE) {
+          continue;
+        }
+
+        var nestedAssignedNodes = Polymer.dom(assignedNodes[i]).queryDistributedElements('*');
```

Comment:
I'd add a short comment here too (or above the whole function, explaining what it does)

Reply:
Done!